optaplanner-examples/src/main/resources/org/optaplanner/examples/cloudbalancing/optional/benchmark/cloudBalancingBenchmarkConfig.xml

```diff
@@ -60,95 +60,95 @@
-        <!--</acceptor>-->
-        <!--<forager>-->
-          <!--<acceptedCountLimit>4</acceptedCountLimit>-->
-        <!--</forager>-->
-      <!--</localSearch>-->
-    <!--</solver>-->
+  <!--<name>Simulated Annealing</name>-->
+  <!--<solver>-->
+  <!--<constructionHeuristic>-->
+  <!--<constructionHeuristicType>FIRST_FIT_DECREASING</constructionHeuristicType>-->
+  <!--</constructionHeuristic>-->
+  <!--<localSearch>-->
+  <!--<unionMoveSelector>-->
+  <!--<changeMoveSelector/>-->
+  <!--<swapMoveSelector/>-->
+  <!--<pillarChangeMoveSelector/>-->
+  <!--<pillarSwapMoveSelector/>-->
+  <!--</unionMoveSelector>-->
+  <!--<acceptor>-->
+  <!--<simulatedAnnealingStartingTemperature>0hard/400soft</simulatedAnnealingStartingTemperature>-->
+  <!--</acceptor>-->
+  <!--<forager>-->
+  <!--<acceptedCountLimit>4</acceptedCountLimit>-->
+  <!--</forager>-->
+  <!--</localSearch>-->
+  <!--</solver>-->
   <!--</solverBenchmark>-->
   <!--<solverBenchmark>-->
-    <!--<name>Late Acceptance</name>-->
-    <!--<solver>-->
-      <!--<constructionHeuristic>-->
-        <!--<constructionHeuristicType>FIRST_FIT_DECREASING</constructionHeuristicType>-->
-      <!--</constructionHeuristic>-->
-      <!--<localSearch>-->
-        <!--<unionMoveSelector>-->
-          <!--<changeMoveSelector/>-->
-          <!--<swapMoveSelector/>-->
-          <!--<pillarChangeMoveSelector/>-->
-          <!--<pillarSwapMoveSelector/>-->
-        <!--</unionMoveSelector>-->
-        <!--<acceptor>-->
-          <!--<lateAcceptanceSize>400</lateAcceptanceSize>-->
-        <!--</acceptor>-->
-        <!--<forager>-->
-          <!--<acceptedCountLimit>4</acceptedCountLimit>-->
-        <!--</forager>-->
-      <!--</localSearch>-->
-    <!--</solver>-->
+  <!--<name>Late Acceptance</name>-->
+  <!--<solver>-->
+  <!--<constructionHeuristic>-->
+  <!--<constructionHeuristicType>FIRST_FIT_DECREASING</constructionHeuristicType>-->
+  <!--</constructionHeuristic>-->
+  <!--<localSearch>-->
+  <!--<unionMoveSelector>-->
+  <!--<changeMoveSelector/>-->
+  <!--<swapMoveSelector/>-->
+  <!--<pillarChangeMoveSelector/>-->
+  <!--<pillarSwapMoveSelector/>-->
+  <!--</unionMoveSelector>-->
+  <!--<acceptor>-->
+  <!--<lateAcceptanceSize>400</lateAcceptanceSize>-->
+  <!--</acceptor>-->
+  <!--<forager>-->
+  <!--<acceptedCountLimit>4</acceptedCountLimit>-->
+  <!--</forager>-->
+  <!--</localSearch>-->
+  <!--</solver>-->
   <!--</solverBenchmark>-->
 
   <!--<solverBenchmark>-->
-    <!--<name>BRANCH_AND_BOUND</name>-->
-    <!--<solver>-->
-      <!--<exhaustiveSearch>-->
-        <!--<exhaustiveSearchType>BRANCH_AND_BOUND</exhaustiveSearchType>-->
-      <!--</exhaustiveSearch>-->
-    <!--</solver>-->
+  <!--<name>BRANCH_AND_BOUND</name>-->
+  <!--<solver>-->
+  <!--<exhaustiveSearch>-->
+  <!--<exhaustiveSearchType>BRANCH_AND_BOUND</exhaustiveSearchType>-->
+  <!--</exhaustiveSearch>-->
+  <!--</solver>-->
   <!--</solverBenchmark>-->
```

Comment:
These changes appear unnecessary.

Reply:
Sorry, automatic formatting via IDE which escaped my attention. Will remove it.